controllers/login.js

```diff
@@ -87,19 +87,51 @@ router.all('/', (req, res, next) => {
 	});
 });
 
+const getIservOauthSystem = (schools) => {
+	for (let schoolIndex = 0; schoolIndex < schools.length; schoolIndex += 1) {
+		const { systems } = schools[schoolIndex];
+		for (let systemIndex = 0; systemIndex < systems.length; systemIndex += 1) {
+			if (systems[systemIndex].type === 'iserv') return systems[systemIndex];
```

Comment:
first record having iserv system because it assumes there is only 1 iserv system?

Reply:
for the generation of the buttons: 
1. in server add system.oauthconfig.enabled flag
2. in this method filter for has oauthconfig and enabled is true
3. respond list off all matching systems
4. generate a list of buttons later of all matching systems (not only the one iserv system)
4.1 the iserv system should be listed again

Reply:
> first record having iserv system because it assumes there is only 1 iserv system?

Yes, for each instance (for now only nbc Niedersachsen) there is only one iServ system.

Reply:
> first record having iserv system because it assumes there is only 1 iserv system?

Yes, since this update, is only for Niedersachsen, and they have only one SSO solution with iServ, it is the only OAuth SSO solution, therefore there will be only one iServ System with the OAuth configuration.